src/js/themes/vanilla.js

```diff
@@ -14,7 +14,7 @@ const statusColors = {
   unknown: '#a8a8a8',
   disabled: '#a8a8a8',
 };
-const darkColors = ['#333333', '#444444', '#555555', '#666666', '#777777', '#999999'];
+const darkColors = ['rgb(51,51,51)', '#444444', '#555555', '#666666', '#777777', '#999999'];
```

Comment:
Why change this to `rgb()`?

Reply:
Previously colorIsDark would not support if we use `rbg`. It would always assume the color was HEX. I've changed this to show that we can support both now. We can revert to `#333333` but I believe it is a good test case to keep.

Reply:
Agreed with the need to test but still think the theme should have hex values.